types/adobe__pdfservices-node-sdk/operation/pdf-properties-operation.d.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * An Operation that is used to fetch properties from an input PDF file. The properties are returned in a JSON file.
+ *
+ * Sample Usage:
+ * <pre class="prettyprint">
+ * <code>const credentials =  PDFServicesSdk.Credentials.serviceAccountCredentialsBuilder()
+ *          .fromFile("pdfservices-api-credentials.json")
+ *          .build(),
+ *       clientContext = PDFServicesSdk.ExecutionContext.create(credentials),
+ *       PDFProperties = PDFServicesSdk.PDFProperties,
+ *       pdfPropertiesOperation = PDFProperties.Operation.createNew(),
+ *       input = PDFServicesSdk.FileRef.createFromLocalFile('test/resources/PDFPropertiesOperationInput.pdf',PDFProperties.SupportedSourceFormat.pdf);
+ *
+ * pdfPropertiesOperation.setInput(input);
+ *
+ * pdfPropertiesOperation.execute(clientContext)
+ * 	.then(result => console.log(result))
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/adobe__pdfservices-node-sdk/operation/pdf-properties-operation.d.ts;
  Line: 17;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.